week02/179.Largest Number.cpp

```diff
@@ -0,0 +1,35 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+///Time complexity O(NlogN)
+///Space complexity O(N)
+
+bool static compare(string a, string b){
+   return (a+b>b+a)? true : false;
+}
+string largestNumber(vector<int>& nums)
+{
+    vector<string>str;

```

Comment:
That was unnecessary. You could have done this string conversion in the compare function.